src/metrics.cpp

```diff
@@ -517,9 +518,10 @@ void ThreadShowMetricsScreen()
 
         // Thank you text
         std::cout << _("Zen is economic freedom. Thanks for running a node.") << std::endl;
-        std::cout << _("仕方が無い") << std::endl;
-        std::cout << _("Shikata ga nai.") << std::endl;
-        std::cout << _("它不能得到帮助") << std::endl << std::endl;
+        if (fPrintToConsole) {
```

Comment:
Probably we could drop the `fPrintToConsole` condition, I'd like to have the version being printed to screen regardless of that option.

Reply:
Done